pages/common/xml-escape.md

```diff
@@ -0,0 +1,16 @@
+# xml escape
+
+> Escape special XML characters.
+> More information: <http://xmlstar.sourceforge.net/doc/xmlstarlet.pdf>.
+
+- Escape special XML characters in a string:
```

Comment:
So what does this command do? Does it return an escaped XML version of that string? Would be great if that was made clear somehow.

Reply:
Yes, it would, but there is no place in the tldr format to show output (or show that a repl is active). Your tldr-lint blocks two lines of example. I'll try to cram a short example of the transformation into the description.

Reply:
xml-escape.md:> Escape special XML characters, e.g. `<a1>` -> `&lt;a1&gt;`.
xml-unescape.md:> Unescape special XML characters, e.g. `&lt;a1&gt;` -> `<a1>`.

Note that these are sensitive characters in markdown.  This works in markdown and the rust tldr client that I have.